cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/StorageSelectorConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.openhouse.cluster.storage.selector;
```

Comment:
can you move this to com.linkedin.openhouse.cluster.storage.configs ?

Reply:
I tried but the unit test fails to compile asking to make the List storageSelectors to be made public due to them being in different packages. I think we should leave the Config in the selector package.